src/content/getusermedia/desktopcapture/extension/content-script.js

```diff
@@ -17,7 +17,7 @@ port.onMessage.addListener(function(msg) {
 });
 
 window.addEventListener('message', function(event) {
-	// We only accept messages from ourselves
```

Comment:
check if your jscs version is different. I've seen that issue with an old version of jshint locally.